app/styles/site-views/search-list.scss

```diff
@@ -0,0 +1,80 @@
+$app-qgreen: rgba(98,144,128,0.25);
+$app-halfgreen: rgba(98,144,128,0.5);
+$app-lightgray: rgb(239,239,239);
+
+.centerdiv-list-wrapper {
+  background-color: $app-lightgray;
```

Comment:
if this is a wrapper that means that all of its children should be nested inside this class in the .scss style sheet: .parent { .child { .grand-child { }}}
the nesting helps keep your classes from being duplicated or changed by subsequent style sheets. please nest when appropriate.

Reply:
ok done.